pkg/otlp/pipeline_validator_serverless.go

```diff
@@ -0,0 +1,24 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+//go:build otlp && serverless
+
+package otlp
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+)
+
+func checkAndUpdateCfg(cfg config.Config, pcfg *PipelineConfig, logsAgentChannel chan *message.Message) error {
+	if HasLogsSectionEnabled(cfg) {
```

Comment:
You can do `cfg.GetBool(config.OTLPLogsEnabled)` and delete the function `HasLogsSectionEnabled`